Makefile

```diff
@@ -70,9 +73,14 @@ $(vmlinux_stripped): $(vmlinux)
 $(vmlinux_binary): $(vmlinux_stripped)
 	$(LINUX_TARGET)-objcopy -O binary $< $@
 
-$(fw_payload): $(opensbi_srcdir) $(vmlinux_binary)
+$(bp_dts):
 	mkdir -p $(opensbi_wrkdir)/platform/blackparrot
-	python $(GENDTS_PY) --ncpus=$(OPENSBI_NCPUS) | dtc -O dtb -o $(opensbi_wrkdir)/platform/blackparrot/blackparrot.dtb
+	python $(GENDTS_PY) --ncpus=$(OPENSBI_NCPUS) --mem-size=$(MEM_SIZE) > $(bp_dts)
+
+$(bp_dtb): $(bp_dts)
+	dtc -O dtb -o $(bp_dtb) $<
```

Comment:
Can you do DTB ?= dtb in this file?

Reply:
If this is a request to add `DTB ?= ...`, yes that can be done if there is a desire to be able to override the gendts.py produced device tree.

If just a question, there is not currently a variable to do that override. The .dts file is generated by gendts.py, which is then fed to dtc to create the binary version (.dtb). Arguments to gendts.py allow us to set number of processors and memory size in the device tree.

Reply:
Sorry typo... DTC ?= dtc

Reply:
ah, lol, got it. I will add that.

Reply:
Similarly PYTHON ?= python